.github/issue_template.md

```diff
@@ -1,3 +1,9 @@
+**Please use the template below for bug reports and minor features/improvements.** 
```

Comment:
I'd end this sentence with "and minor feature requests." (I don't consider a _feature request_ to be the same as a feature.)